bust-a-move-framework/src/main/java/bustamove/game/Cannon.java

```diff
@@ -181,6 +185,18 @@ public final void stepDown() {
     }
 
     /**
+     * Select the side of the cannon which should be updated.
+     * @param side integer of the side.

```

Comment:
I would add a better description of the param. E.g. 0 = turn left. 1 = turn right